.github/workflows/build_run-tests.yml

```diff
@@ -4,29 +4,37 @@ on: [push, pull_request]
 
 jobs:
   on-macos:
-    runs-on: macos-13
+    runs-on: macos-latest
```

Comment:
We pinned this recently due to issues with an update--does it need to change? Can we pin to 14?